frontend/simple/js/events.js

```diff
@@ -3,13 +3,34 @@ import * as Events from '../../../shared/events'
 import store from './state'
 import backend from '../js/backend'
 import {namespace} from '../js/backend/hapi'
+import _ from 'lodash'
 
 export class GroupContract extends Events.HashableGroup {
   static vuex = GroupContract.Vuex({
-    state: { votes: [], payments: [], members: [], invitees: [], profiles: {} },
+    state: { proposals: {}, payments: [], members: [], invitees: [], profiles: {} },
     mutations: {
       HashableGroupPayment (state, {data}) { state.payments.push(data) },
-      HashableGroupVote (state, {data}) { state.votes.push(data) },
+      HashableGroupProposal (state, {data, hash}) { state.proposals[hash] = {...data, for: [], against: []} },
+      HashableGroupVoteForProposal (state, {data}) {
+        if (state.proposals[data.proposalHash]) {
+          state.proposals[data.proposalHash].for.push(data.username)
+          let threshold = Math.ceil(state.proposals[data.proposalHash].percentage * 0.01 * state.members.length)
```

Comment:
I'm seeing this `* 0.01` all over the place. This is repetition, I recommend just storing the actual decimal percentage, and having the *view* convert that as necessary instead.